src/Akeneo/Connectivity/Connection/back/tests/Unit/spec/Domain/Apps/Model/ConnectedAppSpec.php

```diff
@@ -131,4 +131,30 @@ public function it_is_not_a_test_app_by_default(): void
             'is_test_app' => false,
         ]);
     }
+
+    public function is_updates_description_properties(): void
```

Comment:
```suggestion
    public function it_updates_description_properties(): void
```
If you keep this method